codice/ddf
# DDF-4199 Updated AC debugger to 1.5

#### What does this PR do?
It updates the reference to the AC debugger to the latest 1.5 version.

#### Who is reviewing it? 
@ethantmanns 
@jhunzik 
@tyler30clemens 
@kcover 

#### Select relevant component teams: 
@codice/security 
@codice/test 

#### Ask 2 committers to review/merge the PR and tag them here.
@figliold
@tbatie

#### How should this be tested?
1. Build DDF
2. Unzip DDF
3. Replace the `<<ALL FILES>>` for the ssh bundles:
```
grant codeBase "file:/org.apache.karaf.shell.ssh/org.apache.sshd.core" {
    permission java.io.FilePermission "<<ALL FILES>>", "read,write";
}
```
with
```
grant codeBase "file:/org.apache.karaf.shell.ssh/org.apache.sshd.core" {
    permission java.io.FilePermission "${ddf.home.perm}etc${/}keystores${/}-", "read";
    permission java.io.FilePermission "${ddf.home.perm}etc${/}ws-security${/}server${/}-", "read";
    permission java.io.FilePermission "${ddf.home.perm}etc${/}host.key", "read,write";
    permission java.io.FilePermission "${ddf.home.perm}etc${/}branding-ssh.properties", "read";
    permission java.io.FilePermission "${ddf.home.perm}etc${/}org.apache.karaf.shell.cfg", "read";
}
```
4. Connect to DDF using an SSH session: `ssh -p 8101 -o PasswordAuthentication=yes -o PubkeyAuthentication=no -vv  admin@localhost`
5. Start the AC debugger with --debug --continuous --wait
6. Attempt to ingest a file not located under DDF.HOME or the user's home directory via the SSH console
7. Verify the debugger reports on the exception

#### Any background context you want to provide?
The new AC debugger enhances security check to account for inherited/combined access control context when computing.

#### What are the relevant tickets?
[DDF-4199](https://codice.atlassian.net/browse/DDF-4199)

#### Screenshots

#### Checklist:

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.
